calculation/seistech_calc/seistech_calc/disagg/DisaggData.py

```diff
@@ -222,6 +258,24 @@ def __init__(
         self.ensemble = ensemble
         self.im_ensemble = im_ensemble
 
+    @property
+    def fault_disagg_id(self):
+        return self._fault_disagg.set_index(
+            rupture.rupture_id_ix_to_rupture_id(
+                self.im_ensemble.ensemble, self._fault_disagg.index.values
+            ),
+            inplace=False,
+        )
+
+    @property
+    def ds_disagg_id(self):
+        return self._ds_disagg.set_index(
+            rupture.rupture_id_ix_to_rupture_id(
+                self.im_ensemble.ensemble, self._ds_disagg.index.values
+            ),
+            inplace=False,
+        )
```

Comment:
Do these need to be here? Can't we just use DisaggData's properties as they seem the same to me? No need to override them.

Reply:
Agreed, it seems they are identical(BranchDisaggData and EnsembleDisaggData) as well, I reckon it can be moved to the parent's class.

Reply:
THere is a difference between them in the way that they access the ensemble, which is needed for the lookup. However, I should have at least removed the duplication...
But moving it to DisaggData is even better (but it didn't use to have access to the ensemble, and I didn't think it was a good idea to add it, but have changed my mind on that..). Anyways properties will move to DisaggData as suggested.